editors/code/rust.tmGrammar.json

```diff
@@ -167,7 +167,7 @@
             "match": "(mod)\\s+((?:r#(?!crate|[Ss]elf|super))?[a-z][A-Za-z0-9_]*)",
             "captures": {
                 "1": {
-                    "name": "keyword.control.rust"
+                    "name": "storage.type.rust"
```

Comment:
Same as `extern`. @dustypomerleau, @matklad?

Reply:
Same comment, I think this should be `storage.modifier`